src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -69,11 +73,19 @@ public class BigFormat extends AbstractSSTableFormat<BigTableReader, BigTableWri
 {
     private final static Logger logger = LoggerFactory.getLogger(BigFormat.class);
 
+    // Feature flag env variable to use extended TTL up to 2106 with the new oa sstable format. See c14227
+    public static enum TTL_MODE {LEGACY_TTL_2038, COMPATIBILITY, EXTENDED_TTL_2106};
+    @VisibleForTesting
+    public static final String USE_OA_SSTABLE_FORMAT = "cassandra.use_oa_sstable_format";
+    private static final String ttlFeatureFlagValue = System.getProperty(USE_OA_SSTABLE_FORMAT, "").toUpperCase();
```

Comment:
@bereng please move the resolution of this property value to CassandraRelevantProperties.

Reply:
Yep